### PR TITLE
test(e2e): run fio continuously in pod remove

### PR DESCRIPTION
**Pull request by chriswldenyer**
Run fio continuously while pod-loss-induced rebuild tests occur.
Halt fio remotely using the --trigger-file option.

To do: remove the replica hosted on the nexus.

### CODE REVIEW

**Comment by chriswldenyer** [comment]
bors try

**Review by blaisedias — CHANGES_REQUESTED** [review]

File: test/e2e/ms_pod_disruption/disruption_lib.go [hunk 1]
```diff
@@ -109,45 +141,63 @@ func (env *DisruptionEnv) PodLossTest() {
 
 	logf.Log.Info("waiting for pod removal to affect the nexus", "timeout", disconnectionTimeoutSecs)
 	Eventually(func() string {
-		logf.Log.Info("running fio against the volume")
-		_, err := common.RunFio(env.fioPodName, 5, common.FioFsFilename, common.DefaultFioSizeMb)
-		Expect(err).ToNot(HaveOccurred())
-		return common.GetMsvState(env.uuid)
+		time.Sleep(5 * time.Second)
```
[the remark this answers is not in the document]
Comment (chriswldenyer) [reply test/e2e/ms_pod_disruption/disruption_lib.go:144]: Will remove this sleep and the one below. Not sure where they came from.

File: test/e2e/ms_pod_disruption/disruption_lib.go [hunk 2]
```diff
@@ -93,8 +97,36 @@ func getNodes(uuid string) (string, []string) {
 	return toRemove, unusedNodes
 }
 
+// Deploy fio pod, configured to finish and verify, when the trigger-file is created
+func deployHaltableFio(fioPodName string, pvcName string) {
+	podObj := common.CreateFioPodDef(fioPodName, pvcName, common.VolFileSystem, common.NSDefault)
+
+	image := "mayadata/e2e-fio"
```
[the remark this answers is not in the document]
Comment (chriswldenyer) [reply test/e2e/ms_pod_disruption/disruption_lib.go:104]: Ok, will fix.

File: test/e2e/ms_pod_disruption/disruption_lib.go [hunk 3]
```diff
@@ -109,45 +141,63 @@ func (env *DisruptionEnv) PodLossTest() {
...
+		logf.Log.Info("unsuppressing mayastor on unused node", "node", node)
 		UnsuppressMayastorPodOn(node)
 	}
 	logf.Log.Info("waiting for the volume to be repaired", "timeout", repairTimeoutSecs)
 	Eventually(func() string {
-		logf.Log.Info("running fio while volume is being repaired")
-		_, err := common.RunFio(env.fioPodName, 5, common.FioFsFilename, common.DefaultFioSizeMb)
-		Expect(err).ToNot(HaveOccurred())
-		return common.GetMsvState(env.uuid)
+		time.Sleep(5 * time.Second)
```
[the remark this answers is not in the document]
Comment (chriswldenyer) [reply test/e2e/ms_pod_disruption/disruption_lib.go]: fixed

File: test/e2e/ms_pod_disruption/disruption_lib.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (chriswldenyer) [reply test/e2e/ms_pod_disruption/disruption_lib.go]: fixed

File: test/e2e/ms_pod_disruption/disruption_lib.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (chriswldenyer) [reply test/e2e/ms_pod_disruption/disruption_lib.go]: fixed

**Comment by tiagolobocastro** [comment]
@chriswldenyer this should be closed on this repo now?

**Comment by chriswldenyer** [comment]
> @chriswldenyer this should be closed on this repo now?

yes, you are right, closing now,

**Pull request closed without merging**